.circleci/config.yml

```diff
@@ -6,13 +6,12 @@ orbs:
 
 defaults: &defaults
   environment:
-    USE_NATIVE_SOLC: true
     PSQL_ARGS: -p 5432 -h localhost
     NODE_ENV: test
   working_directory: /home/circleci/project
   resource_class: medium
   docker:
-    - image: geoknee/statechannels:12.18-solc0.6.12 # Fast contract compilation with solc installed
```

Comment:
Would you mind creating an issue for using a lighter-weight docker image without `solc`?